app/views/recipes/_recipe.html.erb

```diff
@@ -0,0 +1,34 @@
+
+<div>
+    <h1 class="text-center" > <%= recipe.name %></h1>
+    <div class="d-flex justify-content-between mx-5" id="<%= dom_id recipe %>">
+     <div>
+        <p>Preparation time: <%= recipe.preparation_time %>
+         <% if recipe.preparation_time > 1  %>
+           hours
+           <% else  %>
+          hour
+         <% end %> </p>
+         
+        <p>Cooking time: <%= recipe.cooking_time %> 
+        <% if recipe.cooking_time > 1  %>
+           hours
+           <% else  %>
+          hour
+         <% end %>
+         </p>
```

Comment:
- [ ] Cooking time and preparation time suppose to be in hours not a whole date, So please fix that.
Your App:
![image](https://user-images.githubusercontent.com/68796232/187949268-ade5bd5b-373c-4eae-aa75-093a7bf1c2b9.png)
The given wireframe:
![image](https://user-images.githubusercontent.com/68796232/187949367-897f6ba7-9fae-41f8-9f94-eeca5cc5ef94.png)

Reply:
@Omar-Muhamad, Thanks for your review. We have fixed it.